service-python/tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+import sys
+
+
+def lib_queues(channel):
```

Comment:
How is this being used? Haven't seen something like this so curious as to its purpose